src/nav/MainNavBar.js

```diff
@@ -19,17 +21,17 @@ const moreStyles = StyleSheet.create({
 
 export default class MainNavBar extends React.Component {
   render() {
-    const { openStreamList, onPressPeople } = this.props;
+    const { openStreamList, onPressPeople, color } = this.props;
 
     return (
       <Navigator
         initialRoute={{ name: 'Home', index: 0 }}
         renderScene={(route) =>
           <View style={moreStyles.wrapper}>
-            <View style={styles.navBar}>
-              <Icon style={styles.navButton} name="ios-menu" onPress={openStreamList} />
-              <Title />
-              <Icon style={styles.navButton} name="md-people" onPress={onPressPeople} />
+            <View style={[styles.navBar, { backgroundColor: color }]}>
+              <Icon style={[styles.navButton, { color: foregroundColorFromBackground(color) }]} name="ios-menu" onPress={openStreamList} />
```

Comment:
Since you're computing `foregroundColorFromBackground(color)` twice, it'll be nicer if you set a variable called `textColor` before the `return` statement and use that in both places.